src/ol_infrastructure/components/services/vault.py

```diff
@@ -22,47 +22,58 @@
 DEFAULT_PORT_POSTGRES = 5432
 DEFAULT_PORT_MYSQL = 3306
 DEFAULT_POT_MONGODB = 27017
-SIX_MONTHS = 60 * 60 * 24 * 30 * 6
+SIX_MONTHS = 60 * 60 * 24 * 30 * 6  # noqa: WPS432
 
-class DBEngines(str, Enum):
+
+class DBEngines(str, Enum):  # noqa: WPS600
     """Constraints for valid engine types that are supported by this component."""
     postgres = 'postgresql'
     mariadb = 'mysql'
     mysql = 'mysql'
     mysql_rds = 'mysql_rds'
     mongodb = 'mongodb'
 
+
 class OLVaultDatabaseConfig(BaseModel):
     db_name: Text
     mount_point: Text
     db_admin_username: Text
-    db_admin_password: SecretStr
+    db_admin_password: Text
```

Comment:
Why the change here?

Reply:
Because it was adding unnecessary complexity and I just wanted to remove it as a possible source of error.